classroombroadcast_activity.py

```diff
@@ -0,0 +1,93 @@
+# the Free Software Foundation; either version 2 of the License, or
```

Comment:
This isn't a complete version of the license short version.

Reply:
@chimosky I am not getting you completely....I used command line 2to3 . -w to port all necessary file to python 3 besides this I haven't made any modification

Reply:
Look at other files in the activity and you'll see the difference between the shortened version of the License.

Reply:
@chimosky I have to replace the text you mentioned with the shortened license one given in [setup.py](https://github.com/sugarlabs/classroombroadcast/blob/master/setup.py), am I right?